<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Menu</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Parametry</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="8" column="1">
            <widget class="QSpinBox" name="terrain_x">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Punkt start:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Rozmiar:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="vehicle_num">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Punkt stop:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="3">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="particle_num">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Liczba pojazdów:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QSpinBox" name="terrain_y">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QComboBox" name="map_type">
             <item>
              <property name="text">
               <string>Wzgórza</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Linie</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Skos</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Zęby</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kanion</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Łuk</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Labirynt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Szum</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QSpinBox" name="start_point_y">
             <property name="maximum">
              <number>1000000000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Liczba cząstek:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Iteracje</string>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QSpinBox" name="stop_point_y">
             <property name="maximum">
              <number>1000000000</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QComboBox" name="metric_type">
             <item>
              <property name="text">
               <string>Euklidesowa</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Czebyszewa</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QDoubleSpinBox" name="noise_num">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Metryka:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSpinBox" name="stop_point_x">
             <property name="maximum">
              <number>1000000000</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Szum:</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="3">
            <widget class="QPushButton" name="start_btn">
             <property name="text">
              <string>START</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Typ mapy:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="start_point_x">
             <property name="maximum">
              <number>1000000000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="iteration_num">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="3">
            <widget class="QPushButton" name="generate_map_btn">
             <property name="text">
              <string>Generuj mapę</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Mapa</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="plotwidget" name="map_plot" native="true">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Rezultaty</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Wykres wynikowy</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="plotwidget" name="result_plot" native="true">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Tabela rezultatów</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotwidget</class>
   <extends>QWidget</extends>
   <header>plotwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
